maestro-ios/src/main/java/ios/LocalIOSDevice.kt

```diff
@@ -133,4 +133,8 @@ class LocalIOSDevice(
     override fun isScreenStatic(): Result<Boolean, Throwable> {
         return xcTestDevice.isScreenStatic()
     }
+
+    override fun setPermissions(id: String, permissions: Map<String, String>) {
+        simctlIOSDevice.setPermissions(id, permissions)
```

Comment:
What will happen if a user gives following input:

```
- launchApp:
      permissions: { notifications: deny , all: allow }
```

Reply:
It is expected that all permissions are set to allow except for the notification permission.

Reply:
Validated that the following command on iOS will deny notification permissions, but allow the rest
```
- launchApp:
      permissions: { notifications: deny, all: allow }
```